forge-lda/src/main/java/forge/lda/LDAModelGenerator.java

```diff
@@ -298,99 +282,4 @@ public static boolean topicContains(String card, List<Pair<String, Double>> topi
         return false;
     }
 
-    public static HashMap<String,List<Map.Entry<PaperCard,Integer>>> initializeCommanderFormat(){
```

Comment:
some of these auto refactors look a bit aggressive
why delete useful functions and did you really use Java 8 as target?

Reply:
While useful, initializeCommanderFormat was not used. If we need it back, we can retrieve it from the Git history. Java 8 and Java 11 checks seemed to pass.

Reply:
I restored initializeCommanderFormat. Do you prefer the pull request consisting of the 6 commits above or the pull request consisting of all 11 commits?

Reply:
I think we can just squash merge it later